client/landing/gutenboarding/mixins.scss

```
@@ -98,24 +98,18 @@
 }
 
 @mixin onboarding-block-edge2edge-container {
-	width: calc(
-		100% + #{$gutenboarding-block-margin-mobile} + #{$gutenboarding-block-margin-mobile}
-	);
+	width: calc( 100% + #{$gutenboarding-block-margin-mobile * 2} );
 	margin-left: #{$gutenboarding-block-margin-mobile * -1};
 	margin-right: #{$gutenboarding-block-margin-mobile * -1};
 
 	@include break-small {
-		width: calc(
-			100% + #{$gutenboarding-block-margin-small} + #{$gutenboarding-block-margin-small}
-		);
+		width: calc( 100% + #{$gutenboarding-block-margin-small * 2} );
```

This (and the negative margins below) are not working (I think margin-right can't be negative, not sure though). They're causing some wiggling, as in the page is wider than the viewport. 

See here: https://d.pr/v/2PIUKO

You can overcome this by removing the `margin: 0 20px;` from `plans-modal` and adding this instead:

```css
box-sizing: content-box;
padding: 0 20px
```

Once you do that, you can set this to 
```suggestion
		width: 100%;
```

---

I can't replicate the wiggling and also there's no `margin: 0 20px;` on `.plans-modal`. 

Modifying the box-content & mixin would require retesting also plans table. We can review this again when plans goes through another design iteration.